specification/sql/resource-manager/readme.md

```diff
@@ -31,6 +31,30 @@ tag: package-composite-v4
 
 The following packages may be composed from multiple api-versions.
 
+### Tag: package-composite-v5
+
+These settings apply only when `--tag=package-composite-v5` is specified on the command line.
+
+This section contains the "composite-v5" set of APIs, which is composed from a selection of api-versions that will remain backwards compatible with "v5" clients such as .NET SDK Microsoft.Azure.Management.Sql version 1.44.3.0-preview.
+
+APIs must only be added to this section when the API is publicly available in at least 1 production region and at least 1 generated client has been tested end-to-end.
+
+Differences in v5 (compared to v4):
+
+* Added new API for Azure SQL Server Outbound Firewall Rules
+
+``` yaml $(tag) == 'package-composite-v5'
+input-file:
+- Microsoft.Sql/preview/2021-02-01-preview/OutboundFirewallRules.json
+- Microsoft.Sql/preview/2021-02-01-preview/examples/OutboundFirewallRuleCreate.json
+- Microsoft.Sql/preview/2021-02-01-preview/examples/OutboundFirewallRuleDelete.json
+- Microsoft.Sql/preview/2021-02-01-preview/examples/OutboundFirewallRuleGet.json
+- Microsoft.Sql/preview/2021-02-01-preview/examples/OutboundFirewallRuleList.json
```

Comment:
please remove example file here

Reply:
you can just add `- Microsoft.Sql/preview/2021-02-01-preview/OutboundFirewallRules.json` to v4 tag and 2020-11 tag for test